### ISSUE TITLE
Perceived lag / slow performance with gatsby

### ISSUE THREAD

**Issue by stfndlng** [michelecocuccio/simple-react-lightbox#95]
So I use SRL with gatsby but no matter which settings I chose I can't help but feel a certain lag / slow responsiveness when the lightbox loads or transition between slides. (see video attached)
I wanted to try to disable the spring physics as I suspect it might have to do with that but I can't seem to find an option to disable it.

Any ideas?
Maybe you can add a way to disable the spring feature?

Video (hoping the gif still conveys the issue):
![srl-480](https://user-images.githubusercontent.com/45245610/96616215-d121be80-12f1-11eb-9abe-aee42ab92553.gif)

I use wrapRootElement for the SimpleReactLightbox element in `gatsby-browser.js`:

```
import React from "react"
import SimpleReactLightbox from "simple-react-lightbox"

export const wrapRootElement = ({ element }) => {
  return <SimpleReactLightbox>{element}</SimpleReactLightbox>
}
```

And then dump a bunch of images into the SRLWrapper:
```
<SRLWrapper options={lightboxOptions}>
      <Container>
        <MasonryContainer>
          <Masonry
            breakpointCols={breakpointsObject}
            className="my-masonry-grid"
            columnClassName="my-masonry-grid_column"
          >
            {data.source.edges.map((image, i) => (
                  <Img
                    key={i}
                    fluid={image.node.childImageSharp.fluid}
                    alt={image.node.name}
                  />
                ))}
          </Masonry>
        </MasonryContainer>
      </Container>
    </SRLWrapper>
```

**Comment by michelecocuccio** [comment]
Hello @stfndlng 
I have tested with Gatsby many times and I never had performances issues. Physic is disabled when you used "fade" as option for "slideAnimationType". There could be tons of things that could affect this. Have you tested in another browser? Have you run a performance check using some tools? Have you updated your browser? It's really difficult to narrow this down but I will do some other testing and see if I can replicate the issue.

Thanks

**Comment by stfndlng** [comment]
Thanks for clarifying.
Yes I have tested it on latest Chrome, Firefox and Safari. All packages are also up to date.
This is local btw; using the develop build, will try to see if the production build is different. In the meantime, is it possible to disable any animation/transition completely or could this be added?
Thanks again

**Comment by michelecocuccio** [comment]
@stfndlng is not possibile at the moment I am afraid, I will see what I can do but I can't promise anything right now as I am pretty busy :( I am going to release a new version soon in which I have improved some parts of the code. Although it's wort noticing that this is working fine for me in Gatsby. Also, some other people have been using it with Gatsby and this was working fine. How big are you images? Have you tested with smaller images or just one or two images?

Thanks

**Comment by stfndlng** [comment]
@michelecocuccio I have now also tested this with a production build and with a larger set of images. The images are served in whatever size `gatsby-image` determines is the best for the current resolution. I have also tried different browser sizes to rule out the image size. The result is always the same.
It does not seem to matter how many images or what size they are. I would love to know what settings the other people using gatsby are using so I could try those out. But what I can see is that the transition just makes it look like it's slow.
Sorry if this is not the most accurate debugging description...
